### PR TITLE
Fix/header

### PR DESCRIPTION
**Pull request by borislavkb**
- Add ```h1``` to corresponding routes.

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+import { BsPlusSquare } from "react-icons/bs";
 
 export default function Footer() {
   return (
     <footer className="Footer">
       <nav className="Footer__navigation">
         <NavLink to="/">MyCakes</NavLink>
-        <NavLink to="/add">Plus</NavLink>
+        <NavLink to="/add">
+          <BsPlusSquare size="2rem" />
```
Comment (namirsab) [inline src/components/Footer.js:11]: I'd avoid adding styles both in css and here. You could just style the NavLinks and the icon would inhenrit the font size. Adriana is doing it like this and it works quite well, maybe you could take a look

File: src/components/Header.js [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+          <h1>Single MyCakes</h1>
+        </Route>
+        <Route path="/favs">
+          <h1>Favourites</h1>
+        </Route>
+        <Route path="/add">
+          <h1>Insert a new cake</h1>
+        </Route>
+        <Route exact path="/">
+          <h1>Check the cake!</h1>
```
Comment (namirsab) [inline src/components/Header.js:18]: Maybe here you want to say, `Check the Cakes`?

File: src/components/ItemCard.css [hunk 3]
```diff
@@ -12,6 +12,13 @@
 }
 
 .frame {
-
   border: var(--color-icons) solid;
 }
+
+img {
+  height: 5rem;
+}
```
Comment (namirsab) [inline src/components/ItemCard.css:20]: Let's be a bit more specific here. Styling the `img` element directly would affect every images in the project.
Even though we divide the css in smaller files, the css is not scoped to this component, is loaded and affects globally.

Hence, let's give this `img` a class (e.g. Item-card__image) or something like this instead

File: src/components/ItemCard.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import "./ItemCard.css";
 
-export default function ItemCard() {
+export default function ItemCard({ image }) {
   return (
     <div className="Item-card">
```
Comment (namirsab) [inline src/components/ItemCard.js:5]: Also, as an easy to follow CSS naming convention, you could give the root element of a component always a class name equal to the component name, (in this case, `ItemCard`) and use that as the block in BEM.

File: src/components/ItemCard.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
       <li className="Item-card">
-        <img src="image" alt="description" />
+        <img src={image} alt="description" />
         <h2 className="Item-card__name">Chocolate cake</h2>
```
Comment (namirsab) [inline src/components/ItemCard.js:8]: Why don't you make this dynamic? The item card should receive a prop with the name of the cake no?

File: src/pages/AddNewCake.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import "./AddNewCake.css";
+
+export default function AddNewCake() {
+  return (
+    <form className="form">
```
Comment (namirsab) [inline src/pages/AddNewCake.js:5]: Check my comment above about CSS naming convention

**Review by namirsab — CHANGES_REQUESTED** [review]
@borislavkb good job, small PR and I see you are progressing!
Just some things missing:
* Your description says one thing, but then I see you are doing many other things inside this PR
* There is no issue linked

I've also requested some changes, please take a look :)

File: src/components/Footer.js (hunk 1 above)
Comment (borislavkb) [reply src/components/Footer.js:11]: I agree, using a mix of inline style and css is not a good practice. Will check how Adriana did it, however when I tried to style the NavLinks the icons do not inherit the styles. Also tried to use Context as per instruction on GitHub https://github.com/react-icons/react-icons

File: src/components/Header.js (hunk 2 above)
Comment (borislavkb) [reply src/components/Header.js:18]: Still looking for a proper name for this App and pages :D

File: src/components/ItemCard.css (hunk 3 above)
Comment (borislavkb) [reply src/components/ItemCard.css:20]: I was too eager to see how it looks in LiveServer and missed to apply some proper class. Thx!

File: src/pages/AddNewCake.js (hunk 6 above)
Comment (borislavkb) [reply src/pages/AddNewCake.js:5]: Will implement it. :)

File: src/components/ItemCard.js (hunk 5 above)
Comment (borislavkb) [reply src/components/ItemCard.js:8]: Indeed. I am planning to do this in the next PRs.

**Comment by borislavkb** [comment]
All edits will be applied to ```feat/add-icons```branch due to an issue with the branch flow.

**Pull request closed without merging**